pkgs/by-name/lp/lpac/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  cmake,
+  pkg-config,
+  pcsclite,
+  curl,
+  drivers ? true,
+  libeuicc ? true,
```

Comment:
`drivers` and `libeuicc` should probably be `withDrivers` and `withLibeuicc` so this package does not break if a package named `libeuicc` is added later.